pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -239,6 +240,10 @@ func (configgen *ConfigGeneratorImpl) BuildListeners(node *model.Proxy,
 	push *model.PushContext) []*listener.Listener {
 	builder := NewListenerBuilder(node, push)
 
+	if util.IsIstioVersionGE19(node) {
+		configgen.Plugins = append(configgen.Plugins, metadataexchange.NewPlugin())
```

Comment:
Why does this fix the problem?  don't we still install the 1.9 mx Envoyfilter?

Reply:
we need to remove the yaml section related to that part ... i'm working on adding the missing cluster filter for tcp, once I done that I can remove those 1.9 mx envoyfilter.

Reply:
We probably should target 1.10, considering it is pretty late in the release cycle.